script/openqa-clone-custom-git-refspec

```diff
@@ -27,13 +27,21 @@ fi
 
 curl_github="${curl_github:-"curl${AUTHENTICATED_REQUEST}"}"
 curl_openqa="${curl_openqa:-"curl"}"
+job_list="${job:-"$2"}"
 
 throw_json_error() {
     echo "in contents queried from $1:"
     echo "$2"
     exit 2
 }
 
+extract_urls_from_pr() {
```

Comment:
This function is broken. You will get garbage from the body parsing.

Reply:
See #2787 for an iteration with an improved `jq` call capturing the URL fully and reproducing your case in a unit test.